### DISCUSSION TITLE
Slide Overs

### DISCUSSION THREAD

**Discussion by szerambo** [GuavaCZ/filament-knowledge-base#48] [General]
Where in my files would I use $plugin->slideOverPreviews()? I haven't been able to locate an example of this. Thanks!

**Comment by AFSDARIF** [comment]
try this on Filament AdminPanelProvider

use Filament\View\PanelsRenderHook;
.....
KnowledgeBasePlugin::make()
->helpMenuRenderHook(PanelsRenderHook::TOPBAR_END)
->modalPreviews(true)
->slideOverPreviews(true)
->modalTitleBreadcrumbs(true)
